tools/sst_dump_test.cc

```diff
@@ -520,6 +520,54 @@ TEST_F(SSTDumpToolTest, SstFileDumperMmapReads) {
 
   cleanup(opts, file_path);
 }
+
+TEST_F(SSTDumpToolTest, SstFileDumperVerifyNumRecords) {
```

Comment:
It'd be nice to include cases where `has_from`, `has_to` and `read_num` (ideally renamed to something else) are set.

Reply:
Added tests for them.